lang/pt_BR.json

```diff
@@ -266,7 +266,7 @@
     "in softcore": "em softcore",
     "Additional information for these hashes may be listed on": "Informações adicionais para esses hashes podem estar listadas em",
     "the game's official forum topic": "o tópico oficial do fórum do jogo",
-    "Details on how the hash is generated for each system can be found": "Detalhes sobre como o hash é gerado para cada sistema podem ser encontrados",
+    "Details on how the hash is generated for each system can be found": "Detalhes sobre como o hash é gerado para cada console podem ser encontrados",
```

Comment:
Similar feedback as the other comment about system vs console.

Reply:
It's because of the same reason, if we translate the word system it's not wrong, but if we localize it to Brazil it ends up being wrong, system is used for other purposes, here in Brazil console is used by practically 99% of players, it makes more sense one than the other